automated install/basic-install.sh

```diff
@@ -1833,48 +1833,71 @@ configureFirewall() {
 }
 
 replaceAddCreateRename() {
-    # origVariable="$1"
-    # newVariable="$2"
-    # theFile="$3"
+    # origVariable="${1}"
+    # newVariable="${2}"
+    # theFile="${3}"
```

Comment:
```suggestion
local theFile
theFile="${3}"
```

Reply:
origVariable, newVariable, and theFile are just there for illustration purposes, there not used.  That's why I commented them out.

Reply:
I'm good with using them if you want.

Reply:
Please use them throughout. There shouldn't be any positional variables or iterators that are named with the numeric position or a single letter like `${i}`.

Reply:
I only use ${i} in for loops.  I'll change them to ${var}.